index.bs

```diff
@@ -1160,6 +1161,36 @@ The dictionary SHALL have the following attributes:
 : <dfn for="WebTransportCloseInfo" dict-member>reason</dfn>
 :: The reason for closing the {{WebTransport}}.
 
+## `WebTransportUnidirectionalStreamOptions` Dictionary ##  {#uni-stream-options}
+
+The <dfn dictionary>WebTransportUnidirectionalStreamOptions</dfn> is a
+dictionary of parameters that affect how {{WebTransportSendStream}}s created by
+{{WebTransport/createUnidirectionalStream}} behave.
+
+<pre class="idl">
+dictionary WebTransportUnidirectionalStreamOptions {
+  unsigned long long order;
+};
+</pre>
+
+The dictionary SHALL have the following attributes:
+
+: <dfn for="WebTransportUnidirectionalStreamOptions" dict-member>order</dfn>
+:: An order number that, if provided, opts the created
+   {{WebTransportSendStream}} in to participating in <dfn>strict ordering</dfn>.
+   Bytes currently queued on [=strict ordering|strictly ordered=]
+   {{WebTransportSendStream}}s MUST be sent ahead of bytes currently queued on
+   other [=strict ordering|strictly ordered=] {{WebTransportSendStream}}s
+   created with lower order numbers.
+
+   If no order number is provided, the created {{WebTransportSendStream}}
+   MUST NOT participate in [=strict ordering=], meaning the order in which the
+   user agent sends bytes from it relative to other {{WebTransportSendStream}}s
+   remains [=implementation-defined=].
```

Comment:
Did we not agree on default behaviour?  Even setting expectations would be good.  Something like streams created first will generally be sent first, with a nod toward the starvation text in RFC 9218...

Reply:
I've created a separate PR https://github.com/w3c/webtransport/pull/441 for that, though the two are perhaps inextricably linked, if we want to skip discussing alternatives not on the table.